webapi/src/it/scala/org/knora/sipi/SipiIT.scala

```diff
@@ -103,6 +177,23 @@ object SipiIT extends ZIOSpecDefault {
 }
 
 object MockDspApiServer {
+  object verify {
+    def verifySingleGetRequest(server: WireMockServer, path: String): Boolean =
+      verifyRequest(server, 1, getRequestedFor(urlEqualTo(path))) && server.getAllServeEvents.size() == 1
+    def verifyRequest(server: WireMockServer, amount: Int, requestPattern: RequestPatternBuilder): Boolean =
+      verifyRequest(server, exactly(amount), requestPattern)
+    def verifyRequest(
+      server: WireMockServer,
+      amount: CountMatchingStrategy,
+      requestPattern: RequestPatternBuilder
+    ): Boolean =
+      Try(server.verify(amount, requestPattern)) match {
+        case Failure(e: Throwable) => println(s"\nMockDspApiServer: ${e.getMessage}"); false
```

Comment:
Shouldn't it be logged instead of printed or is this `println` something you forgot to remove?